### ISSUE TITLE
`closure_spacing` not triggering between `{` and `in`

### ISSUE THREAD

**Issue by Vadim-Yelagin** [realm/SwiftLint#3773]
I expect these examples to trigger [closure_spacing](https://realm.github.io/SwiftLint/closure_spacing.html), but they don't:

```swift
foo {_ in bar() }

foo {arg in bar() }

self.foo {[weak self] in self?.bar() }
```

Non-triggering alternatives would be

```swift
foo { _ in bar() }

foo { arg in bar() }

self.foo { [weak self] in self?.bar() }
```

Using Xcode 12.5 (12E262)

**Comment by farshadmb** [comment]
@Vadim-Yelagin  I faced this issue however then I realized we can get rid of it by swiftlint version `0.43.1`. it might be merged with the PR that revokes this ability.

**Comment by jpsim** [comment]
This has since been fixed.

```console
$ cat gh3773.swift
foo {_ in bar() }

foo {arg in bar() }

self.foo {[weak self] in self?.bar() }
$ swiftlint gh3773.swift 
Linting Swift files at paths gh3773.swift
Linting 'gh3773.swift' (1/1)
gh3773.swift:1:5: warning: Closure Spacing Violation: Closure expressions should have a single space inside each brace. (closure_spacing)
gh3773.swift:3:5: warning: Closure Spacing Violation: Closure expressions should have a single space inside each brace. (closure_spacing)
gh3773.swift:5:10: warning: Closure Spacing Violation: Closure expressions should have a single space inside each brace. (closure_spacing)
gh3773.swift:3:6: warning: Unused Closure Parameter Violation: Unused parameter "arg" in a closure should be replaced with _. (unused_closure_parameter)
Done linting! Found 4 violations, 0 serious in 1 file.
```
